.eslintrc

```diff
@@ -14,10 +14,12 @@
   },
   "parser": "babel-eslint",
   "rules": {
+    "import/no-unresolved": "off",
```

Comment:
I'll see if this can be re-enabled if the right `NODE_PATH` is passed to the lint command.